src/SourceBuild/content/eng/tools/PrBaselinePublisher/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.CommandLine;
+using Microsoft.Extensions.Logging;
+
+namespace PrBaselinePublisher;
+
+public class Program
+{
+    public static readonly CliArgument<string> Repo = new("repo")
+    {
+        Description = "The repository to create the PR in.",
```

Comment:
This only supports GitHub rights?
```suggestion
        Description = "The GitHub repository to create the PR in.",
```